src/components/node-list/node-list-items.test.js

```diff
@@ -23,8 +23,8 @@ const ungroupNodes = groupedNodes =>
 
 describe('node-list-selectors', () => {
   describe('getFilteredNodes', () => {
-    const nodes = getGroupedNodes(mockState.animals);
-    const searchValue = 'e';
+    const nodes = getGroupedNodes(mockState.testData);
+    let searchValue = 'bear';
```

Comment:
I'm glad you asked - nice spot 👍 I introduced this change given the current new data arrangement where the `id` in the nodes is a hash that does not contain any reference to the `name` value of `nodes` - by setting the search term as a general `e`, currently it will return nodes with ids that contains 'e' which has no correlation with the string of the `name` and hence causing the tests to break. ( under the previous `animals` arrangement the `id` field is correlated to the `name` field (e.g `id` is something like `task/bear` with name as `bear`, while under the new data type it would be `id: 09f5edeb` and `name: bear`). 

By setting the search term as a more specfic term, it will ensure that only the node with the exact matching term will be returned. I understand this might not allow the testing for a more general search, but this new search term still allows us to test the original search feature of filtering the nodes that contains the search term.